templates/storageclass.yaml

```diff
@@ -0,0 +1,7 @@
+{{- if and ($.Values.global.persistence.storageClassDeploy) (eq $.Values.hosting "aws") }}
```

Comment:
Not sure this is needed like this (storageClassDeploy) as the SC would only be redeployed by helm if it was changed.

Reply:
The storage class changes are normally done at the point of EKS pipeline creation at the point of the efs csi driver instantiation. I wanted to include the definition here so that if a deployment was not done correctly, the user was aware of the changes that might be needed.